spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/stores/GeoJsonStore.java

```diff
@@ -62,6 +63,16 @@ public GeoJsonStore(Context context, SCStoreConfig scStoreConfig) {
         this.scStoreConfig = scStoreConfig;
     }
 
+    public GeoJsonStore(Context context, SCStoreConfig scStoreConfig, SCStyle style) {
```

Comment:
could we have a single constructor here (and in the other stores) and do `this.style = scStoreConfig.getStyle();`?